### PR TITLE
Update de translation

### PR DESCRIPTION
**Pull request by AnnikaStein**
Regular PR (not using the `translations/update` page) because
- there was an error upon submission (I can add the transaction id here or send in private),
- updating one's own branch is more convenient than submitting new PRs, should any changes be required that come up during review sometimes.

Usual workflow via Internationalize, and I cross-checked the original_hash with git diff.

### CODE REVIEW

File: WcaOnRails/config/locales/de.yml [hunk 1]
```diff
@@ -2569,48 +2569,71 @@ de:
...
         #original_hash: bf3a39e
         donation: Zusätzliche Spenden gehen nur an das Organisationsteam
-        #original_hash: 16cf92d
-        card_information: >-
-          Deine Karteninformationen werden nicht an die WCA Website übertragen.
-          Sie werden verschlüsselt an Stripe übertragen, welches die Abwicklung
-          der Zahlung übernimmt und an uns weiterleitet.
+        #original_hash: f7fa263
+        payment_button: >-
+          Deine Zahlungsinformationen gelangen niemals zur WCA Website. Sie
```
Comment (gregorbg) [inline WcaOnRails/config/locales/de.yml:2581]: "erreichen" ist besser. "Gelangen" hat sowas aktives, als würden die Daten wie wuselige kleine Eichhörnchen selbsttätig den Weg zu uns suchen

File: WcaOnRails/config/locales/de.yml [hunk 2]
```diff
@@ -2569,48 +2569,71 @@ de:
...
+          werden verschlüsselt an Stripe übertragen, welches den Zahlungsprozess
```
Comment (gregorbg) [inline WcaOnRails/config/locales/de.yml:2582]: Ist Stripe neutrum? Eher "an Stripe übertragen, wo der Zahlungsprozess abgewickelt wird und anschließend eine Weiterleitung an uns erfolgt"

File: WcaOnRails/config/locales/de.yml [hunk 3]
```diff
@@ -2569,48 +2569,71 @@ de:
...
           Ich würde dem Organisationsteam gerne eine zusätzliche, optionale
           Spende zukommen lassen
+        #original_hash: 4930c06
+        payment_information: Zahlungsinformationen
+        #original_hash: de0fc95
+        payment_button: Zahle deine Gebühren mit Stripe
+        #original_hash: f94d00f
+        ajax_error: WCA Server Error
+        #original_hash: e62c56c
+        stripe_error: Stripe Error
```
Comment (gregorbg) [inline WcaOnRails/config/locales/de.yml:2604]: Ist "Error" als Anglizismus okay in dem Zusammenhang?

File: WcaOnRails/config/locales/de.yml [hunk 4]
```diff
@@ -2569,48 +2569,71 @@ de:
...
-    payment_button_text: Bezahle dein Eintrittsgeld mit Karte
+        #original_hash: a073600
+        stripe_failed: Bitte prüfe die Stripe-Transaktion.
+        #original_hash: a32e40f
+        stripe_not_found: >-
+          Stripe hat eine Transaktions-ID zurückgegeben, die nicht gefunden
+          werden konnte.
+        #original_hash: aaa8ab7
+        stripe_secret_invalid: >-
+          Stripe hat ein Transaktion-Secret zurückgegeben, welches nicht unseren
```
Comment (gregorbg) [inline WcaOnRails/config/locales/de.yml:2631]: Transaktion**s**-Secret

**Review by gregorbg — CHANGES_REQUESTED** [review]
Double nitpicky because not only do you suffer from the usual fact that we have native German WST members but also this time around I have coded the feature that you are (primarily) translating in this PR (so I know _very_ precisely how and where the strings are being used)

**Comment by gregorbg** [comment]
> * there was an error upon submission (I can add the transaction id here or send in private),

Please let us know the error transaction ID through any means that is convenient to you. They are not particularly sensitive so posting them on GitHub is fine; only people with elevated server access can read the logs attached to that ID and the ID itself is generated randomly

**Comment by AnnikaStein** [comment]
> > * there was an error upon submission (I can add the transaction id here or send in private),
> 
> Please let us know the error transaction ID through any means that is convenient to you. They are not particularly sensitive so posting them on GitHub is fine; only people with elevated server access can read the logs attached to that ID and the ID itself is generated randomly

Ok, then it's here: Request ID: edb8932a-441c-43dc-a5ae-ec9635fb6140

Thanks also for all other comments, will modify the branch accordingly :+1: and as always I'm happy to get that feedback from WST members who know the context better than us translators do :)

File: WcaOnRails/config/locales/de.yml (hunk 3 above)
Comment (AnnikaStein) [reply WcaOnRails/config/locales/de.yml]: Geändert zu Fehler, habe das Wort Fehler für Error auch an anderer Stelle wiedergefunden, dann haben wir es einheitlich.

File: WcaOnRails/config/locales/de.yml (hunk 2 above)
Comment (AnnikaStein) [reply WcaOnRails/config/locales/de.yml]: "welches" hatten wir in der vorherigen Iteration der Übersetzung auch schon, also Neutrum hatten wir in der Vergangenheit wohl angenommen. Finde aber i.A. den Vorschlag besser und habe ihn dann einfach komplett übernommen.

File: WcaOnRails/config/locales/de.yml (hunk 1 above)
Comment (AnnikaStein) [reply WcaOnRails/config/locales/de.yml]: Schade um die Eichhörnchen, aber geändert :)

**Review by gregorbg — APPROVED** [review]

**Pull request merged**